mmv1/third_party/terraform/tests/data_source_runtimeconfig_variable_test.go.erb

```diff
@@ -29,15 +29,10 @@ func TestAccRuntimeconfigVariableDatasource_basic(t *testing.T) {
 
 func testAccRuntimeconfigDatasourceVariable(suffix string, name string, text string) string {
 	return fmt.Sprintf(`
-	resource "google_project_service" "default" {
-		service = "runtimeconfig.googleapis.com"
-	}
-
 	resource "google_runtimeconfig_config" "default" {
 		project     = google_project_service.default.project
```

Comment:
There's a trailing reference to the deleted service that needs to be updated.

Reply:
Correction: there are a few of these references in this file.

Reply:
🤦‍♀️  not even something I thought to look for, thanks